src/server/blockchain/MultiWallet.js

```diff
@@ -83,7 +84,8 @@ class MultiWallet {
   }
 }
 
+//exclude celo wallet from tests
 export default new MultiWallet({
   122: AdminWallet, // "main" wallet goes first
-  42220: CeloAdminWallet
+  ...(conf.env === 'test' ? { 42220: CeloAdminWallet } : {})
```

Comment:
@sirpy 
it doesn't helps as CeloAdminWallet have this.ready = this.init() at constructor
look over the commits history there was CELO_ENABLED flag i removed them
see how this was implemented. i made ready prop with getter and calling init() after first access instead from constructoer

Reply:
@sirpy also you did mistake with condition - you're adding celo **if env is test**, should be vice versa
i've fixed: https://github.com/GoodDollar/GoodServer/pull/385/commits/cb931c5d200787c3f76b6acaa4e2e0303e8dd3b4